modules/virt-add-disk-to-vm.adoc

```diff
@@ -28,6 +28,7 @@ Use this procedure to add a virtual disk to a {object}.
 . Select the *Disks* tab.
 . Click *Add Disks* to open the *Add Disk* window.
 . In the *Add Disk* window, specify *Source*, *Name*, *Size*, *Interface*, and *Storage Class*.
+.. Optional: In the *Advanced* drop-down, specify the *Volume Mode* and *Access Mode* for the virtual disk. If you do not specify these parameters, the defaults are used, based on what has been configured in the `kubevirt-storage-class-defaults` ConfigMap.
```

Comment:
s/`In the *Advanced* drop-down`/`From the *Advanced* list` (ISG page 209)

Nit: "the defaults are used, based on what has been configured in the" feels a bit wordy. What about something along these lines:

```
If you do not specify these parameters, the system uses the default values from the `kubevirt-storage-class-defaults` ConfigMap.
```